R/system.R

```diff
@@ -1317,7 +1317,8 @@ scrape_html_table <- function(url, index, heading, encoding = NULL) {
 #' see https://github.com/exploratory-io/tam/issues/1481
 #' @export
 handleLabelledColumns = function(df){
-  is_labelled <- which(lapply(df, class) == "labelled")
+  # check if column class is labelled, haven_labelled, etc
+  is_labelled <- which(stringr::str_detect(lapply(df, class), "labelled"))
```

Comment:
How about just matching with "labelled" and "haven_labelled" if that are all the possibilities we know, to avoid false positive?
Also, some more comment on the context where this is called would help.

Reply:
changed to check both "labelled" and "haven_labelled"

Reply:
Also added the comment that explains why we need this function.